lib/datadog/core/remote/component.rb

```diff
@@ -30,9 +30,14 @@ def initialize(settings, agent_settings)
           @worker = Worker.new(interval: settings.remote.poll_interval_seconds) do
             begin
               @client.sync
+            rescue Client::SyncError => e
+              Datadog.logger.error do
+                "remote worker client sync error: #{e.message} location: #{Array(e.backtrace).first}. skipping sync"
```

Comment:
Minor: I suggest still including the error class, as in the future we may actually catch a subclass of `SyncError` and we'd be throwing away that information